examples/workflows/run_ensemble_model_nudge.py

```diff
@@ -0,0 +1,123 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# %%
+import argparse
+import logging
+import os
+import json
+from functools import partial
+import torch
+from modulus.distributed.manager import DistributedManager
+from earth2mip.inference_ensemble import run_inference, get_initializer
+from earth2mip.ensemble_utils import brown_noise
+from earth2mip.networks import get_model
+from earth2mip.schema import EnsembleRun
+
+
+def apply_gaussian_perturbation(
+     x,
+     time_step,
+     in_channel_names,
+     device,
+     latitute_location,
+     latitute_sigma,
+     longitude_location,
+     longitude_sigma,
+     gaussian_amplitude,
+     modified_channels,
+ ):
```

Comment:
Maybe add a docstirng. It's not immediately clear that this is a perturbation with a guassian shape as opposed to noise.